rule-example.yaml

```diff
@@ -21,7 +21,7 @@
       ignore: true
     - builtin.xml:
         xpath: "//dependencies/dependency"
-        filepaths: '{{ index .poms "filepaths" }}'
+        filepaths: "{{poms.filepaths}}"
```

Comment:
I think the quotes always get converted to `'` because of the parsing. at least, that is what I am seeing in the testing.

Resolving this for now and then we can come back?